cmd/list.go

```diff
@@ -436,6 +436,197 @@ var listNotificationCmd = &cobra.Command{
 	},
 }
 
+var listOrganizationProjectsCmd = &cobra.Command{
+	Use:     "organization-projects",
```

Comment:
I'd look at doing something similar to notifications for organizations based sub commands.

So it would be like `lagoon list organization` and it would present the subcommands like
```
Available Commands:
	projects				List projects within an organization
	groups					List groups within an organizations
	deploytargets		List deploytargets associated to an organization
	users					List users contained within all groups within an organization
```

Reply:
https://github.com/uselagoon/lagoon-cli/blob/main/cmd/list.go#L430-L437 < create the top level command
https://github.com/uselagoon/lagoon-cli/blob/main/cmd/notifications.go#L45C2-L55 < assign the subcommands to it

Reply:
Yep I like it, much cleaner!